### DISCUSSION TITLE
Photo Synchronisation

### DISCUSSION THREAD

**Discussion by dan-gis** [opengisch/QField#2453] [Q&A]
Hello. We're using Qfield for digitizing and we appreciate the possibility to take lots of photos and link them to the features. Works great! Did anybody already test this with qfieldcloud? 
thanks for sharing your experience!

**Comment by mbernasocchi** [comment]
hey @dan-gis yes we did and it worked nicely. but I'm obviously biased ;)

**Reply by dan-gis** [reply]
Thanks! Hope to get access soon so we can try it too! ;-)

**Comment by marioba** [comment]
In order to provide a clearer and more manageable discussion forum and to have all discussions about QField and related products (i.e. QFieldCloud and QFieldSync) in a single place, we transferred all the discussions from the QFieldCloud to the [QField repository](https://github.com/opengisch/QField/discussions).
